### PR TITLE
Actually test the new cloud7 barclamps in gating jobs

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by bmwiedemann** [comment]
How much would that slow down gating runs (including those not related to those new barclamps) ?

**Comment by dirkmueller** [comment]
@bmwiedemann this is gating, not pullrequest testing. pull requests are already deploying this.

File: scripts/scenarios/cloud7/cloud7-2nodes-default.yml [hunk 1]
```diff
@@ -151,6 +156,40 @@ proposals:
...
       - @@controller@@
+- barclamp: magnum
+  attributes:
+    cert:
+      cert_manager_type: barbican
+  deployment:
+    elements:
+      magnum-server:
+      - @@controller@@
+- barclamp: manila
```
Comment (aplanas) [inline scripts/scenarios/cloud7/cloud7-2nodes-default.yml]: duplicated in L55

**Comment by toabctl** [comment]
Retriggering the gate...

**Comment by jdsn** [comment]
retriggering gating again ...

**Pull request merged**